azure-pipelines-official.yml

```diff
@@ -265,10 +265,13 @@ extends:
 
               - task: MicroBuildSigningPlugin@4
                 inputs:
-                  signType: $(SignType)
+                  signType: ${{ parameters.SignType }}
                   zipSources: false
                   feedSource: https://dnceng.pkgs.visualstudio.com/_packaging/MicroBuildToolset/nuget/v3/index.json
-                condition: and(succeeded(), in(variables['SignType'], 'test', 'real'))
+                  # Set ConnectedPMEServiceName if the build is a CI build, otherwise it is not needed
+                  ${{ if eq(variables['Build.Reason'], 'IndividualCI') }}:
```

Comment:
rolled https://github.com/dotnet/roslyn/pull/80090 into this one since they're both hitting the same test issues anyway